doc/usermanual/CMakeLists.txt

```diff
@@ -9,8 +9,10 @@ message(STATUS ${IMAGES_PATHS})
 
 if(BUILD_USERMANUAL)
 	add_custom_target(usermanual ALL)
+	add_custom_target(api ALL)
```

Comment:
Maybe call it something else, these target names are going to the global namespace, and something like `api` could be needed for other things. And it's not that obvious what it does. How about `apimanual` instead?